lib/dry/transaction/step.rb

```diff
@@ -51,13 +55,13 @@ def call(input, continue = RETURN)
       end
 
       def with_broadcast(args)
-        broadcast :step, step_name, *args
+        publish('step', step_name: step_name, args: args)
```

Comment:
Is it dry-events' suggested style to use strings for event names? I see they're in the docs as such but I wonder if it makes sense to use symbol naming in simple cases like this? /cc @solnic

Reply:
Symbols and strings are fine, whatever fits better